upcloud/client/client.go

```diff
@@ -184,9 +203,12 @@ func (c *Client) PerformRequest(request *http.Request) ([]byte, error) {
 
 // Returns the base URL to use for API requests
 func (c *Client) getBaseURL() string {
-	urlVersion, _ := semver.Make(DefaultAPIVersion)
+	url := upcloud.GetEnvOrDefault(APIBaseURLEnv, DefaultAPIBaseURL)
+	version := upcloud.GetEnvOrDefault(APIVersionEnv, DefaultAPIVersion)
```

Comment:
not sure if we really need two different environment variables to control one thing (eg. the endpoint url?) could we not just pass in the version in the url if we're overriding the base url anyhow?